### PR TITLE
Redbubble Icon

### PR DESCRIPTION
**Pull request by TomerADev**
I added an entry for the [Redbubble](https://www.redbubble.com/explore/) icon!

### CODE REVIEW

File: icons/redbubble.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg aria-labelledby="simpleicons-redbubble-icon" role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title id="simpleicons-adobephotoshop-icon">Redbubble icon</title><path d="M16.633 16.324h-3.199a.321.321 0 0 1-.32-.322V7.974a.32.32 0 0 1 .32-.32H16.4c2.226 0 2.693 1.31 2.693 2.408 0 .636-.169 1.14-.504 1.511.816.337 1.256 1.096 1.256 2.194 0 1.601-1.201 2.557-3.212 2.557m-4.644 0H5.345a.32.32 0 0 1-.32-.322V7.974a.32.32 0 0 1 .32-.32h3.103c1.939 0 3.096 1.043 3.096 2.791 0 1.163-.585 2.077-1.527 2.448l2.21 2.897a.322.322 0 0 1-.24.533M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12c6.628 0 12-5.373 12-12S18.63 0 12.001 0"/></svg>
```
Comment (birjolaxew) [inline icons/redbubble.svg]: `id="simpleicons-adobephotoshop-icon"` should probably be `id="simpleicons-redbubble-icon` ;)

Comment (TomerADev) [reply icons/redbubble.svg]: Fixed!

**Comment by birjolaxew** [comment]
:+1: Thanks for the contribution! It is much appreciated.

**Comment by TomerADev** [comment]
No problem!

**Pull request merged**